src/main/java/edu/kis/powp/jobs2d/command/gui/backlog/BackLogManagerWindowCommandChangeObserver.java

```diff
@@ -0,0 +1,17 @@
+package edu.kis.powp.jobs2d.command.gui.backlog;
+import edu.kis.powp.observer.Subscriber;
+
+public class BackLogManagerWindowCommandChangeObserver implements Subscriber {
+
+    private BackLogManagerWindow backLogManagerWindow;
```

Comment:
Try to redesign the history component to be independent of the GUI. (e.g. we might want to change GUI or use history from another component without accessing GUI).
There should be a separate class representing the history.